.github/workflows/lint-codebase.yaml

```diff
@@ -22,7 +22,46 @@ jobs:
           VALIDATE_ALL_CODEBASE: false
           DEFAULT_BRANCH: main
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          FILTER_REGEX_EXCLUDE: '.*doc/.*|.*github/.*'
+          FILTER_REGEX_EXCLUDE: ".*doc/.*|.*github/.*"
+
+  check-helm-resources-changed:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          # Full git history is needed to get the diff with the main branch
+          fetch-depth: 0
+
+      - name: Get Helm Chart resources diff
+        id: check-helm-resources-diff
+        continue-on-error: true
+        run: |
+          export DIFF=$(git diff --unified=0 origin/main origin/${GITHUB_HEAD_REF} chart/values.yaml | grep -E "[0-9]+Mi|[0-9]+m")
+          if [ -n "$DIFF" ]; then
+            echo "resources-changed=true" >> $GITHUB_OUTPUT
+          else
+            echo "resources-changed=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Comment on PR if Helm Chart resources changed
+        uses: peter-evans/create-or-update-comment@v2
+        if: steps.check-helm-resources-diff.outputs.resources-changed == 'true'
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            :warning: Helm Chart resources have changed. Please make sure to update the documentation accordingly. You can find more information [here](https://cartodb.getoutline.com/doc/helm-chart-yJ3vxi6icN#h-deployment-resources).
+
+      - name: Slack notification
```

Comment:
I think that this is unnecesary (or only required in a merge to main)

Reply:
I've changed it to send the notification just when pushing to main 🙇🏼‍♂️ IMHO it's necessary as a comment in the PR can be omitted, but with that notification we have no excuses 😂